pkg/e2e/harness_runner/harness_runner.go

```diff
@@ -0,0 +1,38 @@
+package harness_runner
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/onsi/ginkgo/v2"
+	viper "github.com/openshift/osde2e/pkg/common/concurrentviper"
+
+	"github.com/openshift/osde2e/pkg/common/alert"
+	"github.com/openshift/osde2e/pkg/common/config"
+	"github.com/openshift/osde2e/pkg/common/helper"
+	"github.com/openshift/osde2e/pkg/common/prow"
+)
+
+var _ = ginkgo.Describe("[Suite: harnessess] Test Harness", func() {
+	defer ginkgo.GinkgoRecover()
+	h := helper.New()
```

Comment:
helpers need to be created in `BeforeAll` or `BeforeEach` to prevent it from being created before Ginkgo actually runs the test

Reply:
Add the helper to a BeforeAll